sdk/identity/src/token_credentials/default_credentials.rs

```diff
@@ -13,11 +14,7 @@ pub struct DefaultCredentialBuilder {
 impl DefaultCredentialBuilder {
     /// Create a new `DefaultCredentialBuilder`
     pub fn new() -> Self {
-        Self {
-            include_cli_credential: true,
-            include_managed_identity_credential: true,
-            include_environment_credential: true,
-        }
+        Self::default()
```

Comment:
This effectively sets the `include_cli_credential`, `include_managed_identity_credential`, & `include_environment_credential` to `false`, which is but #526.